src/lib/tokens/balancer-fetcher.ts

```diff
@@ -247,6 +246,7 @@ export const decodeBalanceAndAllowanceMultiResult = (
 export const getBalances = async (
   multiv2: MultiWrapper,
   reqs: BalanceRequest[],
+  blockNumber: number | string = 'latest',
```

Comment:
I think this is type safer:
`  blockNumber: number | 'latest' = 'latest',`

Reply:
True